torch/_torch_docs.py

```diff
@@ -1465,6 +1465,69 @@ def merge_dicts(*dicts):
     tensor([ 0.5000, -0.4702, -0.4599,  0.5000])
 """.format(**common_args))
 
+add_docstr(torch.complex,
+           r"""
+complex(real, imag, *, out=None) -> Tensor
+
+Constructs a complex tensor with its real part equal to :attr:`real` and its
+imaginary part equal to :attr:`imag`.
+
+.. math::
+    \text{out} = \text{real} + \text{imag} \cdot i
```

Comment:
Our complex numbers are a + bj, not a + bi.

Reply:
That said, I don't know if you need a math example here. The center dot notation may not be super clear, anyway.